src/v2/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebarExtraLinks.jest.tsx

```diff
@@ -91,10 +91,7 @@ describe("ArtworkSidebarExtraLinks", () => {
     })
     it("displays conditions of sale link that opens conditions of sale page", () => {
       expect(wrapper.find('a[children="Conditions of Sale"]').length).toBe(1)
-      wrapper
-        .find('a[children="Conditions of Sale"]')
-        .at(0)
-        .simulate("click")
+      wrapper.find('a[children="Conditions of Sale"]').at(0).simulate("click")
```

Comment:
Sorry for the large diff here. Looks like there was some prettier update that caused the format changes. Probably safe to ignore.